graph.py

```diff
@@ -0,0 +1,18 @@
+from cProfile import label
```

Comment:
Is this needed? It looks like it might've been an accidental auto-import from L14

Reply:
Seems to be.